src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -2233,6 +2248,29 @@ protected CSharpSyntaxNode GetBindableParentNode(CSharpSyntaxNode node)
             return bindableParent;
         }
 
+        internal override Symbol RemapSymbolIfNecessaryCore(Symbol symbol)
+        {
+            Debug.Assert(symbol is LocalSymbol ||
+                         symbol is ParameterSymbol ||
+                         symbol is MethodSymbol { MethodKind: MethodKind.LambdaMethod });
+            Debug.Assert(Compilation.NullableSemanticAnalysisEnabled);
+            EnsureNullabilityAnalysisPerformedIfNecessary();
+
+            if (_lazyRemappedSymbols is null)
+            {
+                return symbol;
+            }
+
+            if (_lazyRemappedSymbols.TryGetValue(symbol, out var remappedSymbol))
```

Comment:
`if (_lazyRemappedSymbols?.TryGetValue(...) == true)`

Reply:
Same comment about definite assignment.

---
In reply to: [331619521](https://github.com/dotnet/roslyn/pull/38769#discussion_r331619521) [](ancestors = 331619521)